### ISSUE TITLE
get Accessibility threshold from user

### ISSUE THREAD

**Issue by yaelAmitay** [TechnionYP5777/SmartCity-Accessibility#160]
the navigation won't pass through streets with Accessibility level below some threshold. the threshold should be given by the user. we need to get it some way/
I'll probably open a small window asking the user to insert the desired threshold.

**Comment by Kolikant** [comment]
suggestion:
both elaborate search and navigation need the desired accessability as a known variable.
maybe we shout of the user profile.

he won't have different accessability for navigation and searching and would probably stick to the same level?

**Comment by kapalex** [comment]
It should be a setting that is set for each search/navigation, not something that the is a user setting
If a user wants to search with a different accessibility constraint each time it should be normal, and not something tedious he needs to change in the setting
@yaelAmitay solution of just opening a window and asking seems better to me

**Comment by Kolikant** [comment]
That is the existing one already in search, thought encapsulating it would be a good call
Also, having a default value in settings seems higly reasonable, as users would most of the time use the same value for they have the same accessability needs

**Comment by kapalex** [comment]
It doesn't from a stand point we never defined what each accessibility provides, so while a certain place could have a rating of 3, which be fine for a user, but another place with a rating of 4 wouldn't be
The user will want to search once with a 4 rating requirement, and if he doesn't find something that fits his needs he will lower his standard and search for a 3

**Comment by yaelAmitay** [comment]
So we decided to pop a window to ask right? This is my current planning...

Yael

‫ב-12 בינו׳ 2017, בשעה 22:37, ‏‏Alexander Kaplan ‏<notifications@github.com> כתב/ה:‬

**Comment by yaelAmitay** [comment]
done.

**Comment by Kolikant** [comment]
very confusing interface. when the popup closes, the navigation window is still open allowing naviagate tp be pressed again, no indecation is given as to the fact that navigation is accuring.
wither close the naviagation window, dissallow activating navigation from the same window again or give indecation that navigation is calculating

**Comment by yaelAmitay** [comment]
this is the location frame you are talking about. i'm not sure if we want to close it!
I think the interface is very clear, you pick a threshold and thats it. 
If you want to close the location frame fine by me but this has nothing to do with this issue or with the navigation at all!

**Comment by Kolikant** [comment]
I can tell you that it isn't clear from the fact that I tried to use it and wasn't sure if it's running at all, causing me to run navigation a second time.

i press navigation, a pop up arrives and then is gone, no idea if its running. it can be confusing, it WAS confusing for me dissallow pressing navigation again (grey the button to show naviagtion is running) is the simplest solution

**Comment by yaelAmitay** [comment]
then let's open a different issue about letting the user know the nvaigation is working.
instead of reopening this issue which is not relevant...

**Comment by Kolikant** [comment]
its not as if the issues are a sacred thing to be used in one particular way. put it where you think its convenient for you

**Comment by yossigil** [comment]
It is better slightly for progress monitoring to open a new issue
